android/build.gradle

```diff
@@ -13,20 +13,26 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
+        minSdkVersion safeExtGet('minSdkVersion', 21)
```

Comment:
We're aiming API 18 on `sentry-android-ndk`. For `sentry-android` only it's API 14

Reply:
you may keep 16, and do this on manifest -> https://github.com/getsentry/sentry-android/blob/fd6413b92ed97efdecccc1fa6d8abef27fb1c816/sentry-sample/src/main/AndroidManifest.xml#L7

so you support non-ndk from 16 to 20 and with ndk support from 21 on

Reply:
no changes needed on manifest, minSdkVersion=16 just works :)